composer.json

```diff
@@ -19,7 +19,7 @@
     "keywords": [ "x690", "x.690", "x.509", "x509", "asn1", "asn.1", "ber", "der", "binary", "encoding", "decoding" ],
 
     "require": {
-        "php": "~7.1.0 || ~7.2.0 || ~7.3.0 || ~7.4.0 || ~8.0.0 || ~8.1.0"
+        "php": "~7.1.0 || ~7.2.0 || ~7.3.0 || ~7.4.0 || ~8.0.0 || ~8.1.0 || ~8.2.0"
```

Comment:
`>=7.1`?

Reply:
hi there, I created this PR because one of our apps was not allowing the update since web-token packages depend on it. thx for your work there! 😊

I assume the review comments are more relevant for the maintainer of this package, I'll leave this up for him if you don't mind, but I can adapt it any time of course

Reply:
> web-token packages depend on it

That won't be true in a few days.